plugins/MauticCrmBundle/Integration/ConnectwiseIntegration.php

```diff
@@ -779,6 +796,22 @@ public function populateLeadData($lead, $config = [])
                 }
             }
 
+            if ($integrationKey === 'company' && isset($fields['company']) && !empty($fields['company']['value'])) {
```

Comment:
`isset($fields['company'])` doesn't have to be there. It's already covered in `!empty($fields['company']['value'])`. But apart from longer if condition it won't hurt.